srcpkgs/elog/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'elog'
+pkgname=elog
+_elogrel=1
```

Comment:
Why is this line here? Maybe should be part of the version? Definitely not up at pkgname,version,revision.

Reply:
is there a way to better reflect the upstream's naming scheme which looks like $pkgname-$version_$revision?

Reply:
See my work in `slurm-wlm`
```
pkgname=slurm-wlm
version=18.08.1.1
revision=3
_distver="${version//./-}"
```

Reply:
There the version scheme is consistent `-` instead of `.` but there are shell features to make these changes.